### PR TITLE
Modified Wallet history UI in sim exchange

### PR DESCRIPTION
**Pull request by nitinrgupta**
##### Description

Updated the UI of wallet history tab based on the new designs

##### Checklist

- [x] Linter status: 100% pass
- [x] Changes don't break existing behavior
- [ ] Tests coverage hasn't decreased

##### Affected core subsystem(s)

UI

##### Refers/Fixes

Fixes: #339

### CODE REVIEW

File: src/components/SimExchange/Wallet/Columns.js [hunk 1]
```diff
@@ -1,58 +1,99 @@
...
+                    theme="filled"
+                    onClick={() => copyTextToClipboard(text.details.hash)}
+                  />
+                </div>
+              </Col>
+            </Row>
+            <Button
+              type="primary"
+              className="popover-action"
+              href={`${getEtherscanUrl('rinkeby')}/address/${
```
Comment (nitinrgupta) [inline src/components/SimExchange/Wallet/Columns.js]: @travisdmathis @eswarasai any idea what is the best way to determine the network type over here to generate the etherscan URL?

Comment (eswarasai) [reply src/components/SimExchange/Wallet/Columns.js]: @nitinrgupta -- You should do something similar as here -- https://github.com/MARKETProtocol/dApp/blob/develop/src/components/DeployContract/Steps.js#L776

Comment (nitinrgupta) [reply src/components/SimExchange/Wallet/Columns.js]: @eswarasai Hmmm yeah! But I don't have the props inside table datasource :/ 
Is there any global methods to determine the network type? 🤔

Comment (nitinrgupta) [reply src/components/SimExchange/Wallet/Columns.js]: Oh nvm. I found another way. It was easier than I thought. Thanks for the help @eswarasai!

**Pull request merged**